[9.0][FIX]stock_auto_move: assign automatic procurement to only auto moves without proc.

zakiuu opened the pull request:
**Actual behavior:** With the `stock_auto_move` module, moves that are marked _automatic_ are processed automatically without the interaction of the users. In the mean time, those moves are assigned to an `automatic procurement` blindly. The author of the module has a reason doing so, and it has been explained in #180.
**The issue:** Procurement group have a strong _functional semantic_ (For example: when doing a reception in two steps, a procurement group is generated to tie up the 2 created picking for each one of the steps.).

This PR attends to fix this issue, by adding an _automatic procurement_ to only moves that aren't assigned to a procurement group.

zakmak88 committed:
[IMP]stock_auto_move: auto move from push rules

zakmak88 committed:
[FIX]stock_auto_move: assign automatic procurement to only auto moves without proc.

zakiuu commented:
This PR is based on #358

lmignon reviewed (approved):
@zakiuu changes in https://github.com/OCA/stock-logistics-workflow/pull/364/commits/a274769ce37bd5cfd477fc7dfbd1d65179a50ac1 LGTM... We must wait for the merge of #358

zakmak88 committed:
[REF]stock_auto_move: remove override of _apply method

* The `_apply` method should not be overridden. In case of three step
operation with the last step set to auto move. The override of the
`_apply` method was causing the second move to be automatic as well.

gurneyalex commented:
@npiganeau can you check this PR?

npiganeau commented on stock_auto_move/models/stock_location_path.py:
```diff
@@ -1,15 +1,23 @@
+             "automatically processed as soon as the products are available. "
+             "This can be useful for situations with chained moves where we "
+             "do not want an operator action."
+    )
```
Although I do understand the main subject of the PR and agree with it, I do not understand why you modify the `stock.location.path` code logic. It seems to me unrelated to the subject of this PR. Maybe it could be in another PR ?

zakiuu replied:
Hi @npiganeau thanks for the review, but this PR is based on #359  and that's the reason why you see those commits here. I had to do so because the migration of the module wasn't merged yet

npiganeau replied:
Ok, I understand.

npiganeau commented on stock_auto_move/views/stock_location_path.xml:
```diff
@@ -0,0 +1,18 @@
+        </field>
+    </record>
+
+</odoo>
```
Same remark here
